falcon/__init__.py

```diff
@@ -50,7 +50,7 @@
 from falcon.response import Response, ResponseOptions  # NOQA
 
 
-ASGI_SUPPORTED = _sys.version_info.minor > 5
+ASGI_SUPPORTED = _sys.version_info >= (3, 6)
```

Comment:
let's support python 4 🚀

Reply:
Lol I already had this type of comment elsewhere :)